plugins/doc_fragments/client_inst_opts.py

```diff
@@ -49,6 +49,14 @@ class ModuleDocFragment(object):
     type: dict
     default: {}
 
+  flatten_nested:
+    description:
+      - Updates flatten_nested setting on session before running
+        query
+      - Only runs the update if set to 0
+    type: int
+    default: 1
+
```

Comment:
```suggestion
  flatten_nested:
    description:
      - Sets the C(flatten_nested) setting on session before running the query.
    type: int
    choices: [0, 1]
    version_added: '0.5.0'
```
- this is the `clickhouse_client` related argument, so please move it to plugins/modules/clickhouse_client.py
- let's skip the default and let users specify it explicitly?